app/Commands/LamboCommand.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace App\Commands;
+
+use LaravelZero\Framework\Commands\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+abstract class LamboCommand extends Command
+{
+    public function handle()
+    {
+        app()->bind('console', function () {
+            return $this;
+        });
+
+        app()->singleton('console-writer', function () {
+            return new class($this->input, $this->output) {
```

Comment:
Can we re-create this as an actual class?

Reply:
Done.